platform/fabric/core/generic/config/config.go

```diff
@@ -87,6 +87,26 @@ func (c *Config) TLSRootCertFile() string {
 	return c.configService.GetString("fabric." + c.prefix + "tls.rootCertFile")
 }
 
+func (c *Config) KeepAliveClientInterval() time.Duration {
+	return c.configService.GetDuration("fabric." + c.prefix + "tls.keepalive.client.interval")
+}
+
+func (c *Config) KeepAliveClientTimeout() time.Duration {
+	return c.configService.GetDuration("fabric." + c.prefix + "tls.keepalive.client.timeout")
+}
+
+func (c *Config) KeepAliveServerInterval() time.Duration {
+	return c.configService.GetDuration("fabric." + c.prefix + "tls.keepalive.server.interval")
+}
+
+func (c *Config) KeepAliveServerTimeout() time.Duration {
+	return c.configService.GetDuration("fabric." + c.prefix + "tls.keepalive.server.timeout")
+}
+
+func (c *Config) KeepAliveServerMinInterval() time.Duration {
+	return c.configService.GetDuration("fabric." + c.prefix + "tls.keepalive.server.minInterval")
+}
```

Comment:
What would happen if the configuration does not contain the entry? Would `c.configService.GetDuration` just panic or do we have default values somewhere? Maybe directly from `core_template.go`?

Reply:
good point, let's test :)

Reply:
actually, it just return the default value the type of variable can get. Duration gets 0, being an int64.